Content.Shared/_RMC14/Shields/XenoShieldSystem.cs

```diff
@@ -62,7 +63,11 @@ public void OnDamage(Entity<XenoShieldComponent> ent, ref DamageModifyEvent args
             RemoveShield(ent, ent.Comp.Shield);
         }
         else
+        {
+            if (HasComp<ProjectileComponent>(args.Tool) && args.Damage.DamageDict.ContainsKey("Piercing"))
```

Comment:
"Piercing" should at least be a static readonly field at the top of this file so that it can be validated to exist
`private static readonly ProtoId<DamageTypePrototype> _shieldSoundDamageType = "Piercing";`
```suggestion
            if (HasComp<ProjectileComponent>(args.Tool) && args.Damage.DamageDict.ContainsKey("Piercing"))
```